cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -101,3 +153,25 @@ func (pe *ProbeEvent) initOrErrorEvent() {
 		klog.Error(err)
 	}
 }
+
+// generateUUID creates a new UUID based on the algorithm proposed in
+// https://github.com/openebs/openebs/pull/2666
+func generateUUID(bd blockdevice.BlockDevice) (string, bool) {
+	if bd.DeviceAttributes.DeviceType == libudevwrapper.UDEV_PARTITION {
+		klog.Info("device is a partition, using partition UUID:", bd.PartitionInfo.PartitionEntryUUID)
```

Comment:
add devpath also in the log so that we can correlate..

Reply:
Done.